pages/common/chafa.md

```diff
@@ -0,0 +1,25 @@
+# chafa
+
+> Image printing in the terminal.
+> See also: `catimg`, `pixterm`.
+> More information: <https://hpjansson.org/chafa>.
+
+- Render an image or animated GIF directly in the terminal:
+
+`chafa {{path/to/file}}`
+
+- Render an image with 24-bit [c]olor:
+
+`chafa -c full {{path/to/file}}`
+
+- Improve image rendering with small color palettes using dithering:
+
+`chafa -c 16 --dither ordered {{path/to/file}}`
+
+- Render an image in the terminal to appear pixelated:
+
+`chafa --symbols vhalf {{path/to/file}}`
+
+- Render a monochrome image with only Braille characters:
```

Comment:
```suggestion
- Render a monochrome image with only braille characters:
```

Not sure whether this should be capitalized, from a little research, I think it shouldn't be.

Reply:
Nice. I hadn't realized.